lms/djangoapps/verify_student/tests/test_models.py

```diff
@@ -90,39 +89,11 @@ def mock_software_secure_post_unavailable(url, headers=None, data=None, **kwargs
     raise requests.exceptions.ConnectionError
 
 
-class TestVerification(TestCase):
```

Comment:
Renamed `TestVerificationBase` and moved to `__init__.py` so that it can be imported as `from verify_student.tests import TestVerificationBase`